docs/decision_record.md

```diff
@@ -6,6 +6,52 @@
 
 [TOC]
 
+## 2021-12-01 - Endpoint names and structure
+
+### Decision
+
+The endpoint names will be:
+
+- `GET /service-info` for GA4GH service info
+- `GET /collection/{digest}` for retrieving a sequence collection
+- `GET /comparison/{digest1}/{digest2}` for comparing two collections in the database
+- `POST /comparison/{digest1}/` for comparing one database collection to a local user-provided collection.
```

Comment:
Is a trailing slash needed here on the POST?

Reply:
No. good point.